docs/guide/context.md

```diff
@@ -0,0 +1,191 @@
+---
+id: context
+title: "\U0001F9E0 Context"
```

Comment:
Context might be ambiguous name as we already have `Ctx` page. I think we need to find a different name

Reply:
i agree, maybe something with go context
or context.context

@coderabbitai suggest a good name

Reply:
@ReneWerner87 updated